tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from app import create_app, db
+from flask.signals import request_finished
+from app.models.planet import Planet
+
+
+@pytest.fixture
```

Comment:
Nice work writing up the boiler plate code needed for the app and client fixtures and also writing a custom fixture for saving some planets to the db!